_troubleshooting/mst/invalid-relay-state.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Invalid RelayState when attempting an IdP-initiated
+auth flow
```

Comment:
```suggestion
title: Invalid RelayState when attempting an IdP-initiated auth flow
```

Reply:
Can't have a line break here because the new line will be parsed as a new YAML key-value pair. If you want a multi-line string, you can use either `>` or `|`, where the former collapses whitespace and the latter preserves it.

I also usually add a `-` to strip the ending newline.

For example:
```yaml
title: >-
  Invalid RelayState when attempting an IdP-initiated
  auth flow
```